client/src/components/Draftail/blocks/ImageBlock.js

```diff
@@ -13,18 +12,16 @@ class ImageBlock extends Component {
   constructor(props) {
     super(props);
 
-    this.changeAlt = this.changeAlt.bind(this);
+    this.onEditEntity = this.onEditEntity.bind(this);
   }
 
-  changeAlt(e) {
-    const { block, blockProps } = this.props;
-    const { editorState, onChange } = blockProps;
-
-    const data = {
-      alt: e.target.value,
-    };
+  onEditEntity(e) {
+    const { blockProps } = this.props;
+    const { entity, onEditEntity } = blockProps;
 
-    onChange(DraftUtils.updateBlockEntity(editorState, block, data));
+    e.preventDefault();
+    e.stopPropagation();
+    onEditEntity(entity);
   }
```

Comment:
You need to add tests for this new method.

Reply:
I removed this method as was no longer required.